src/main/java/frc/robot/Constants.java

```diff
@@ -71,18 +81,62 @@
             public static final int kSpinnerMotorPort = 9; // motor for the wheel that spins the control panal
 
             public static final int kColorSensorPort = 0; // Color sensor that senses the colors on the control panel
+
+            /*values the color sensor values to detect the colors on control panel
+            red*/
```

Comment:
This probably should not use a multi line comment as `red` is not a continuation of the last statement.
```suggestion
            // values the color sensor values to detect the colors on control panel
            // red
```